std/experimental/logger/core.d

```diff
@@ -2100,7 +2100,7 @@ unittest // default logger
 unittest
 {
     import std.file : deleteme, remove;
-    import core.memory : destroy;
```

Comment:
`destroy` is defined in `object` module, and each modules imports `object` privately. So this is just wrong.